rpm/gmime.spec

```diff
@@ -39,25 +34,16 @@ Internet Mail Extension (MIME). The devel-package contains header files
 to develop applications that use libgmime.
 
 %prep
-%setup -q -n %{name}-%{version}/%{name}
+%autosetup -p1 -n %{name}-%{version}/%{name}
 
-# gmime-2.5.8-gpg-error.patch
-%patch0 -p1
-# disable-gtkdoc.patch
-%patch1 -p1
-# Add-automake-1.16.1-support.patch
-%patch2 -p1
-%__cp $RPM_SOURCE_DIR/gpgme.m4 m4/
 
 %build
-AUTOGEN_SUBDIR_MODE=1 %autogen || true
-
+./autogen.sh
 %configure  \
     --disable-gtk-doc \
```

Comment:
Extra poins for making explicit disables/enables for the current build configs: https://github.com/sailfishos-mirror/gmime/blob/6bff5288f9c108160dd56dd20a9c6608aa55f791/configure.ac#L679